README.md

```diff
@@ -7,61 +7,168 @@
 
 * Explain that by making parts of their code *conditional*, they can execute different pieces of code based on certain conditions.
 * Explain that an *if* statement will execute a set of statements **only** if that condition is *true*. 
+* Explain that an *if* statement can provide an alternative set of statements, known as an *else clause*, for situations when the *if* condition is *false*.
+* Create/chain multiple *if-else* statements together to consider additional clauses:
+
+
+
+## Outline / Notes
+
+In the previous lesson we covered Boolean values (namely ````true```` and ````false````) along with the various operations we can perform that result in a Boolean value (namely ````<, ==, <=, >, >=, !=````).
+For example:
+
+````Swift
+3 < 5    // false
+12 >= 12  // true
+let greedoFiredFirst  // true
+! greedoFiredFirst  // false
+````
+
+At this point you may find Booleans interesting but are wondering what they're actually good for.  If that's the case then you're in luck because that's *exactly* what this lesson is about.
 
-```swift
-var age = 19
+So far you've covered computer code that's of the form of directives and declarations; that is, "do this".  For example:
 
-if age < 21 {
-    print("You're not allowed to drink!")
+````Swift
+print("hello")   // hello
+// directive to print "hello"
+
+var x = 5 * 8
+// declaration of variable x
+
+let i = 12 >= 9
+// declaration of constant i
+````
+
+Most programs, however, need an additional component, and that component is Conditionals.  Conditionals are how you tell the program to take different behavior based upon conditions or results.  The simplest form of Conditional is just to do something if an expression is true.  We can express "if class is hard then study more" like this:
+
+````Swift
+if classIsHard {
+	studyMore()
 }
+````
 
-// prints "You're not allowed to drink!"
-```
+We can also specify an "else" clause.  For example, we can express "if class is hard then study more otherwise catch up on sleep" ike this:
 
-* Explain that an *if* statement can provide an alternative set of statements, known as an *else clause*, for situations when the *if* condition is *false*.
+````Swift
+if classIsHard() {
+	studyMore()
+} else {
+	catchUpOnSleep()
+}
+````
 
-```swift
-var myHeight = 50
+Are you begining to see the pattern?  Let's look at another one, maybe something more like an actual program, like "if the file exists then open it, otherwise tell the user it can't be found".  Let's see what it would look like in code:
 
-if myHeight > 60 {
-    print("You're free to go on the ride, enjoy!")
+````Swift
+let doesFileExist = findFileNamed("testfile")
```

Comment:
I like the idea of being slightly more explicit here:

---

```swift
func findFileNamed(name: String) -> Bool {
    
    return name == "Prom Photo"
    
}

let doesFileExist = findFileNamed("Prom Photo")

if doesFileExist == true {
    
    // open the file
    print("We were able to open the file.")
    
} else {
    
    // report the error
    print("There was an error!")
    
}

// prints "We were able to open the file."
```

Here we created a function `findFileNamed(_:)` which takes in one argument of type `String` with a return type of `Bool`. If the `String` passed into this function is equal to 'Prom Photo' then this will return `true`, if it's not, it will return `false`.

On the next line, we create a constant named `doesFileExist` which equals the result of the call to the function `findFileNamed(_:)`. As you can see, we're passing in "Prom Photo" to this function, which will result in this function returning back `true` to the caller which means `doesFileExist` is of type `Bool` and its value is set to `true`. If we passed in any other `String` that wasn't "Prom Photo" then this function would have returned back to us a `false` value.

Slightly different than above, we are being explicit in our `if` statement where we are writing code that looks like this:

```swift
if doesFileExist == true
// this reads like a sentence, if doesFileExist is equal to true the { } curly braces after the if statement and execute that code
```

or..

```swift
if doesFileExist
// this reads like a sentence, if doesFileExist is equal to true the { } curly braces after the if statement and execute that code
```

They are both identical, there is no difference. The way that is preferred, is by not being explicit - there's no need for us to write `== true`` but be aware that you should then be naming your variables and constants with names that reflect that they contain `Bool` values.